src/tests/unit/api-next/get-all-open-orders.test.ts

```diff
@@ -0,0 +1,162 @@
+          "ocaType": 3,
+          "eTradeOnly": false,
+          "firmQuoteOnly": false,
+          "nbboPriceCap": 1.7976931348623157e+308,
+          "parentId": 0,
+          "triggerMethod": 0,
+          "volatility": 1.7976931348623157e+308,
+          "volatilityType": 0,
+          "deltaNeutralOrderType": "None",
+          "deltaNeutralAuxPrice": 1.7976931348623157e+308,
+          "deltaNeutralConId": 0,
+          "deltaNeutralSettlingFirm": "",
+          "deltaNeutralClearingAccount": "",
+          "deltaNeutralClearingIntent": "",
+          "deltaNeutralOpenClose": "?",
+          "deltaNeutralShortSale": false,
+          "deltaNeutralShortSaleSlot": 0,
+          "deltaNeutralDesignatedLocation": "",
+          "continuousUpdate": 0,
+          "referencePriceType": 0,
+          "trailStopPrice": 1.7976931348623157e+308,
+          "trailingPercent": 1.7976931348623157e+308,
+          "basisPoints": 1.7976931348623157e+308,
+          "basisPointsType": 1.7976931348623157e+308,
+          "scaleInitLevelSize": 0,
+          "scaleSubsLevelSize": 1.7976931348623157e+308,
+          "scalePriceIncrement": 1.7976931348623157e+308,
+          "hedgeType": "",
+          "optOutSmartRouting": false,
+          "clearingAccount": "0",
+          "clearingIntent": "",
+          "notHeld": false,
+          "algoStrategy": "0",
+          "solicited": false,
+          "whatIf": false,
+          "randomizeSize": false,
+          "randomizePrice": false,
+          "conditions": [],
+          "adjustedOrderType": "None",
+          "triggerPrice": 1.7976931348623157e+308,
+          "lmtPriceOffset": 1.7976931348623157e+308,
+          "adjustedStopPrice": 1.7976931348623157e+308,
+          "adjustedStopLimitPrice": 1.7976931348623157e+308,
+          "adjustedTrailingAmount": 1.7976931348623157e+308,
+          "adjustableTrailingUnit": 0,
+          "softDollarTier": {
+            "name": "",
+            "value": "",
+            "displayName": ""
+          },
+          "cashQty": 0,
+          "dontUseAutoPriceForHedge": true,
+          "isOmsContainer": false,
+          "discretionaryUpToLimitPrice": false,
+          "usePriceMgmtAlgo": true
+        },
+        "orderState": {
+          "status": OrderStatus.Submitted,
+          "initMarginBefore": 1.7976931348623157e+308,
+          "maintMarginBefore": 1.7976931348623157e+308,
+          "equityWithLoanBefore": 1.7976931348623157e+308,
+          "initMarginChange": 1.7976931348623157e+308,
+          "maintMarginChange": 1.7976931348623157e+308,
+          "equityWithLoanChange": 1.7976931348623157e+308,
+          "initMarginAfter": 1.7976931348623157e+308,
+          "maintMarginAfter": 1.7976931348623157e+308,
+          "equityWithLoanAfter": 1.7976931348623157e+308,
+          "commission": 1.7976931348623157e+308,
+          "minCommission": 1.7976931348623157e+308,
+          "maxCommission": 1.7976931348623157e+308,
```

Comment:
So these should be numbers or undefined now? @mfrener 

cc @iamfotx

Reply:
Yes, on IBApiNext there shall be no MAX_NUMBER but should be undefined (in contrast to IBApi).
Think we should add a helper function for this, as we will need it on other place too.
Something like:
```
undefineMaxAttributes(obj: unknown): void {
  Object.keys(obj).forEach(key => {
     if (typeof (<Record<String,Number>>obj)[key]=== 'number') {
         (<Record<String,Number>>obj)[key] = undefineMax(obj);
     }
  });
}
```
and use it to undefine MAX_NUMBER 's on the given object(s)